components/admin-panel/sections/accounting/AccountingCategoryForm.tsx

```diff
@@ -0,0 +1,171 @@
+    onSubmit: opts.onSubmit,
+  });
+
+  const { setFieldValue, values } = formik;
+  React.useEffect(() => {
+    if (values.kind.value !== AccountingCategoryKind.EXPENSE) {
+      setFieldValue('expensesTypes', null);
+    }
+  }, [setFieldValue, values.kind]);
+
+  return formik;
+}
+
+export const AccountingCategoryKindI18n = defineMessages({
+  [AccountingCategoryKind.EXPENSE]: {
+    id: 'Expense',
+    defaultMessage: 'Expense',
+  },
+  [AccountingCategoryKind.ADDED_FUNDS]: {
+    id: 'AccountingCategory.Kind.ADDED_FUNDS',
+    defaultMessage: 'Added Funds',
+  },
+  [AccountingCategoryKind.CONTRIBUTION]: {
+    defaultMessage: 'Contribution',
+  },
+});
+
+export function mapKindToOption(
+  intl: IntlShape,
+  kind: AccountingCategoryKind,
+): { value: AccountingCategoryKind; label: string } {
+  return {
+    value: kind,
+    label: intl.formatMessage(AccountingCategoryKindI18n[kind]),
+  };
+}
+
+type AccountingCategoryFormProps = {
+  formik: ReturnType<typeof useFormik<FormValues>>;
+};
+
+export function AccountingCategoryForm(props: AccountingCategoryFormProps) {
+  const intl = useIntl();
+
+  const accountingCategoryKindOptions = [
+    {
+      value: AccountingCategoryKind.EXPENSE,
+      label: intl.formatMessage(AccountingCategoryKindI18n[AccountingCategoryKind.EXPENSE]),
+    },
+    {
+      value: AccountingCategoryKind.ADDED_FUNDS,
+      label: intl.formatMessage(AccountingCategoryKindI18n[AccountingCategoryKind.ADDED_FUNDS]),
+    },
+    {
+      value: AccountingCategoryKind.CONTRIBUTION,
+      label: intl.formatMessage(AccountingCategoryKindI18n[AccountingCategoryKind.CONTRIBUTION]),
+    },
+  ];
+
+  const expenseTypeOptions = Object.values(omit(ExpenseType, ExpenseType.FUNDING_REQUEST)).map(t => ({
+    value: t,
+    label: i18nExpenseType(intl, t),
+  }));
+
+  return (
+    <React.Fragment>
+      <StyledInputField name="kind" required label={intl.formatMessage({ defaultMessage: 'Category kind' })} mt={3}>
+        <StyledSelect
+          {...props.formik.getFieldProps('kind')}
+          inputId="kind"
+          options={accountingCategoryKindOptions}
+          required
+          width="100%"
+          maxWidth={500}
+          onChange={({ value }) =>
+            props.formik.setFieldValue(
+              'kind',
+              accountingCategoryKindOptions.find(c => c.value === value),
+            )
+          }
+        />
+      </StyledInputField>
+      <StyledInputField name="name" required label={intl.formatMessage({ defaultMessage: 'Category name' })} mt={3}>
+        <StyledInput
+          {...props.formik.getFieldProps('name')}
+          required
+          width="100%"
+          maxWidth={500}
+          maxLength={60}
+          onChange={e => props.formik.setFieldValue('name', e.target.value)}
+        />
+      </StyledInputField>
+      <StyledInputField
+        required={false}
+        name="friendlyName"
+        label={intl.formatMessage({ defaultMessage: 'Friendly name' })}
```

Comment:
Let's re-use the existing string to not force contributors to re-translate it (you'll need to `git checkout main lang/*` before updating the languages)

```suggestion
        label={intl.formatMessage({ id: 'AccountingCategory.friendlyName', defaultMessage: 'Friendly name' })}
```